README.md

```diff
@@ -4,10 +4,36 @@
 The Braket Algorithm Library provides Amazon Braket customers with pre-built implementations of prominent quantum algorithms and experimental workloads as ready-to-run example notebooks.
 
 ---
+### Braket algorithms
+
 Currently, Braket algorithms are tested on Linux, Windows, and Mac.
 
 Running notebooks locally requires additional dependencies located in [notebooks/textbook/requirements.txt](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/requirements.txt). See notebooks/textbook/README.md for more information.
 
+| Algorithm | Notebook | References | 
+| ----- | ----- | ----- |
+| Bell's Inequality     | [Bells_Inequality.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Bells_Inequality.ipynb)     | [Bell1964](https://journals.aps.org/ppf/abstract/10.1103/PhysicsPhysiqueFizika.1.195), [Greenberger1990](https://doi.org/10.1119/1.16243)     |
```

Comment:
Minor suggestion - target URLs which link to files within the current repo should ideally be [relative paths](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files). For example:
```suggestion
| Bell's Inequality     | [Bells_Inequality.ipynb](notebooks/textbook/Bells_Inequality.ipynb)     | [Bell1964](https://journals.aps.org/ppf/abstract/10.1103/PhysicsPhysiqueFizika.1.195), [Greenberger1990](https://doi.org/10.1119/1.16243)     |
```